### ISSUE TITLE
Describe how to set environment variables in README

### ISSUE THREAD

**Issue by Laurin-W** [activitypods/app-boilerplate#8]

**Comment by srosset81** [comment]
Isn't the link to the tutorial https://docs.activitypods.org/tutorials/create-your-first-social-app/ enough ?

I would prefer to avoid having the same instructions in double.

**Comment by Laurin-W** [comment]
Good point. I was trying to set up the boilerplate without docker-compose the other day and I ran in to quite a lot of issues of how the `.env` files depend on each other. But expanding on this in the docs and a few lines in the .env files themselves should be fine.
Also I ran into issues of version compatibility with docker-compose. My debian server did not support the latest which caused some trouble there too.

**Comment by srosset81** [comment]
> Also I ran into issues of version compatibility with docker-compose. My debian server did not support the latest which caused some trouble there too.

Maybe we should specify the expected Docker compose version ? It's true that, in the latest versions, there is no more `version` properties in the YAML files.
